bitrise.yml

```diff
@@ -149,18 +170,26 @@ workflows:
             #!/bin/bash
             set -e
 
-            if [ $BITRISE_APP_DIR_PATH != $BITRISE_EXPORTED_SIMULATOR_BUILDS_PATH_EXPECTED ] ; then
```

Comment:
instead: `BITRISE_EXPORTED_SIMULATOR_BUILDS_PATH_EXPECTED ` please add the expected word into the `BITRISE_APP_DIR_PATH`, to make it clear